plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/validator/LabelKeyValidator.java

```diff
@@ -24,12 +26,14 @@
 	
 	public static final int SUBDOMAIN_MAXLENGTH = 253;
 	private static final Pattern SUBDOMAIN_REGEXP = Pattern.compile("^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$");
+	private Collection<String> readonlykeys;
```

Comment:
@adietish This probably should be a set but for now there is only one value.  I dont anticipate haveing many others.